src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject

```

Comment:
The `@Inject` annotation is not necessary here. It is used to inject dependencies, and in this case, there are none. Please remove it.